tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -1773,6 +1787,606 @@ function ( $location ) use ( &$redirect_url ) {
+		$this->assertSame( '1', $query['wc_stripe_cs'] ?? null );
+		$this->assertSame( WC_Stripe_UPE_Payment_Gateway::ID, $query['wc_payment_method'] ?? null );
+		$this->assertSame( (string) $order->get_id(), $query['order_id'] ?? null );
+		$this->assertNotEmpty( $query['_wpnonce'] ?? null );
+		$this->assertSame( 1, wp_verify_nonce( $query['_wpnonce'], 'wc_stripe_process_checkout_session_redirect_nonce' ) );
+
+		// The session id must have been linked to the order.
+		$this->assertSame(
+			'cs_test_disambig',
+			WC_Stripe_Order_Helper::get_instance()->get_stripe_checkout_session_id( wc_get_order( $order->get_id() ) )
+		);
+	}
```

PHPMD (2.15.0)

[error] test_process_payment_with_checkout_session_returns_disambiguation_url accesses the super-global variable $_POST. (undefined)

(Superglobals)